src/internal/middleware.js

```diff
@@ -31,6 +29,8 @@ export default function sagaMiddlewareFactory(options = {}) {
           const store = createStore(reducer, applyMiddleware(sagaMiddleware))
           sagaMiddleware.run(saga, ...args)
       `)
```

Comment:
the long msg was introduced with breaking changes of 0.10. perhaps we could get rid of it now and just keep the short message